docs/test.ts

```diff
@@ -105,7 +105,11 @@ tracer.init({
     obfuscatorKeyRegex: '.*',
     obfuscatorValueRegex: '.*',
     blockedTemplateHtml: './blocked.html',
-    blockedTemplateJson: './blocked.json'
+    blockedTemplateJson: './blocked.json',
+    eventTracking: {
+      enabled: true,
```

Comment:
This property is not being used, so we should remove from the test and from the `index.d.ts`